ext/random/php_random.h

```diff
@@ -195,14 +195,29 @@ static inline uint64_t php_random_pcgoneseq128xslrr64_rotr64(php_random_uint128_
 }
 # endif
 
-# define php_random_bytes_throw(b, s) php_random_bytes((b), (s), 1)
-# define php_random_bytes_silent(b, s) php_random_bytes((b), (s), 0)
-# define php_random_int_throw(min, max, result) php_random_int((min), (max), (result), 1)
-# define php_random_int_silent(min, max, result) php_random_int((min), (max), (result), 0)
-
 PHPAPI zend_result php_random_bytes(void *bytes, size_t size, bool should_throw);
 PHPAPI zend_result php_random_int(zend_long min, zend_long max, zend_long *result, bool should_throw);
 
+static inline zend_result php_random_bytes_throw(void *bytes, size_t size)
+{
+	return php_random_bytes(bytes, size, 1);
```

Comment:
Could switch to `true`/`false` since we're already touching these.

Reply:
Makes sense, done.